experimenter/experimenter/nimbus_ui_new/views.py

```diff
@@ -62,7 +64,17 @@ class NimbusChangeLogsView(NimbusExperimentViewMixin, DetailView):
 
 class NimbusExperimentsListView(NimbusExperimentViewMixin, FilterView):
     queryset = (
-        NimbusExperiment.objects.all()
+        NimbusExperiment.objects.annotate(
+            sort_end_date=Coalesce(
+                F("_end_date"),
+                ExpressionWrapper(
+                    Coalesce(F("proposed_release_date"), F("_start_date"))
+                    + F("proposed_duration"),
+                    output_field=DateField(),
+                ),
+            )
+        )
```

Comment:
Yeah also here we should just be sorting in python space on teh `computed_end_date` field so that we don't have multiple parallel definitions of what 'computed_end_date' means.